### PR TITLE
iOS SDK doc for EUID

### PR DESCRIPTION
**Pull request by genwhittTTD**
- Add iOS SDK file. Some mods needed still.
- Add lead-ins from Publisher Overview and SDK Summary
- Update sidebar.
Note updates are needed to code samples, references to code elements, URLs, all of which have UID2 in them.

### CODE REVIEW

File: docs/overviews/overview-publishers.md [hunk 1]
```diff
@@ -103,13 +104,13 @@ For a detailed summary of web integration options, see [Web Integration Overview
...
 
-<!-- ### Mobile Integrations
+### Mobile Integrations
 
 The following resources are available for publisher integrations supporting mobile devices.
 
 | Integration Type| Documentation | Content Description |
 | :--- | :--- | :--- |
--->
+| iOS | [SDK for iOS Reference Guide](../sdks/sdk-ref-ios.md) | An SDK that facilitates the process of generating or establishing client identity using UID2 and retrieving EUID tokens for publishers that need to support iOS apps. |
```
Comment (caroline-ttd) [inline docs/overviews/overview-publishers.md:113]: I'm not sure whether we should use "UID2" or "EUID" in this context: "generating or establishing client identity using UID2."

File: docs/sdks/sdk-ref-ios.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+Add the following entry in your `Podfile`:
+
+```
+pod 'UID2', '~> 1.2'
+```
+
+## Usage Guidelines
+
+The **UID2Manager** singleton is the primary developer API for the SDK for iOS. It is responsible for storing, refreshing, and retrieving the EUID Identity including the EUID token.
```
Comment (caroline-ttd) [inline docs/sdks/sdk-ref-ios.md:106]: UID2Manager  -> EUIDManager

File: docs/sdks/sdk-ref-ios.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+
+For iOS, the `UID2Manager` is initialized automatically the first time it is accessed. You can configure it to support automatic or manual refresh capabilities.
```
Comment (caroline-ttd) [inline docs/sdks/sdk-ref-ios.md:108]: UID2Manager -> EUIDManager

File: docs/sdks/sdk-ref-ios.md [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+### Variables
+
+The following variables are available as part of the UID2Manager API:
+
+- [identity](#identity)
+- [identityStatus](#identitystatus)
+
+#### identity
+
+The Identity variable stores and returns the current `UID2Identity` data object being managed by the SDK.
```
Comment (caroline-ttd) [inline docs/sdks/sdk-ref-ios.md:192]: That is correct, we are also using `UID2Identity` for EUID.

File: docs/sdks/sdk-ref-ios.md [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+UID2Manager.shared.setIdentity(_ identity: UID2Identity)
+```
+
+Get the EUID token (advertising token) to pass to the Advertising SDK (for ad request or bidstream use):
+
+```js
+UID2Manager.shared.getAdvertisingToken()
+```
+
+## UID2Manager API
```
Comment (caroline-ttd) [inline docs/sdks/sdk-ref-ios.md:145]: UID2Manager -> EUIDManager

File: docs/sdks/sdk-ref-ios.md [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+
+This section includes the functions and variables that are part of the UID2Manager API.
```
Comment (caroline-ttd) [inline docs/sdks/sdk-ref-ios.md:147]: UID2Manager -> EUIDManager

File: docs/sdks/sdk-ref-ios.md [hunk 7]
```diff
@@ -0,0 +1,197 @@
...
+
+If the current EUID Identity is valid, this function returns the EUID token (advertising token).
+
+#### setAutomaticRefreshEnabled()
+
+Toggle for automatic refresh functionality.
+
+### Variables
+
+The following variables are available as part of the UID2Manager API:
```
Comment (caroline-ttd) [inline docs/sdks/sdk-ref-ios.md:185]: UID2Manager -> EUIDManager

File: docs/sdks/sdk-ref-ios.md [hunk 8]
```diff
@@ -0,0 +1,197 @@
...
+UID2Manager.shared.getAdvertisingToken()
+```
+
+## UID2Manager API
+
+This section includes the functions and variables that are part of the UID2Manager API.
+
+### Functions
+
+The following functions are available as part of the UID2Manager API:
```
Comment (caroline-ttd) [inline docs/sdks/sdk-ref-ios.md:151]: UID2Manager -> EUIDManager

File: docs/sdks/sdk-ref-ios.md [hunk 9]
```diff
@@ -0,0 +1,197 @@
...
+
+
+## Code Samples
+
+The following code samples provide examples of performing specific activities relating to managing EUID with the SDK for iOS.
+
+Generate an initial EUID Identity (for instructions, see [Client-Side Integration Guide for Mobile](../guides/integration-mobile-client-side#configure-the-uid2-mobile-sdk)):
+
+```js
+UID2Manager.shared.generateIdentity(
```
Comment (caroline-ttd) [inline docs/sdks/sdk-ref-ios.md:126]:
```
EUIDManager.shared.generateIdentity(
    _ identity: IdentityType,
    subscriptionID: String,
    serverPublicKey: String,
    appName: String? = nil
)
```

File: docs/sdks/sdk-ref-ios.md [hunk 10]
```diff
@@ -0,0 +1,197 @@
...
+    _ identity: IdentityType,
+    subscriptionID: String,
+    serverPublicKey: String,
+    appName: String? = nil
+)
+```
+Set the Initial EUID Identity (for instructions, see [Client-Server Integration Guide for Mobile](../guides/integration-mobile-client-server#configure-the-uid2-mobile-sdk)):
+
+``` javascript
+UID2Manager.shared.setIdentity(_ identity: UID2Identity)
```
Comment (caroline-ttd) [inline docs/sdks/sdk-ref-ios.md:136]: `EUIDManager.shared.setIdentity(_ identity: UID2Identity)`

File: docs/sdks/sdk-ref-ios.md [hunk 11]
```diff
@@ -0,0 +1,197 @@
...
+```
+
+Get the EUID token (advertising token) to pass to the Advertising SDK (for ad request or bidstream use):
+
+```js
+UID2Manager.shared.getAdvertisingToken()
```
Comment (caroline-ttd) [inline docs/sdks/sdk-ref-ios.md:142]: `EUIDManager.shared.getAdvertisingToken()`

File: docs/sdks/sdk-ref-ios.md [hunk 12]
```diff
@@ -0,0 +1,197 @@
...
+- [generateIdentity()](#generateidentity)
+- [setIdentity()](#setidentity)
+- [resetIdentity()](#resetidentity)
+- [refreshIdentity()](#refreshidentity)
+- [getAdvertisingToken()](#getadvertisingtoken)
+- [setAutomaticRefreshEnabled()](#setautomaticrefreshenabled)
+
+#### generateIdentity()
+
+Generate am EUID Identity using <Link href="../ref-info/glossary-uid#gl-dii">Directly identifying information (DII)</Link>. For details, see [Configure the EUID mobile SDK](../guides/integration-mobile-client-side.md#configure-the-uid2-mobile-sdk) in the *Client-Side Integration Guide for Mobile*.
```
Comment (caroline-ttd) [inline docs/sdks/sdk-ref-ios.md:161]: We might need to update the link ../guides/integration-mobile-client-side.md#configure-the-uid2-mobile-sdk to an EUID-specific one once it's created. Same as some other links.

File: docs/overviews/overview-publishers.md (hunk 1 above)
Comment (genwhittTTD) [reply docs/overviews/overview-publishers.md:113]: Definitely a miss on my part. Fixed. Thanks!

File: docs/sdks/sdk-ref-ios.md (hunk 2 above)
Comment (genwhittTTD) [reply docs/sdks/sdk-ref-ios.md:106]: Fixed, all instances, thx.

File: docs/sdks/sdk-ref-ios.md (hunk 6 above)
Comment (genwhittTTD) [reply docs/sdks/sdk-ref-ios.md:147]: Done, all.

File: docs/sdks/sdk-ref-ios.md (hunk 8 above)
Comment (genwhittTTD) [reply docs/sdks/sdk-ref-ios.md:151]: Done, all.

File: docs/sdks/sdk-ref-ios.md (hunk 12 above)
Comment (genwhittTTD) [reply docs/sdks/sdk-ref-ios.md:161]: Ah yes thx. I missed that. For now, commented out all those links to a doc that isn't on EUID yet.

**Review by caroline-ttd — APPROVED** [review]

File: docs/sdks/sdk-ref-ios.md [hunk 13]
```diff
@@ -0,0 +1,198 @@
...
+| Name | Location | Dependency Rule                         |
+| :--- | :--- |:----------------------------------------| 
+| uid2-ios-sdk | `git@github.com:IABTechLab/uid2-ios-sdk.git` | Up to next major version: 1.7.0 < 2.0.0 |
+
+### Installing with CocoaPods
+
+Add the following entry in your `Podfile`:
+
+```
+pod 'UID2', '~> 1.2'
```
Comment (caroline-ttd) [inline docs/sdks/sdk-ref-ios.md:101]: Probably it should be `pod 'UID2', '~> 1.7'`

Comment (genwhittTTD) [reply docs/sdks/sdk-ref-ios.md:101]: Fixed. TYSM.

**Pull request merged**